### PR TITLE
Mssindhurid azure sdk js

### PR DESCRIPTION
**Pull request by mssindhurid**
Initial release of azure-security-attestation Beta version for JS

### CODE REVIEW

**Review by LarryOsterman — APPROVED** [review]

**Comment by mssindhurid** [comment]
@czubair  - I see JS SDK PR - https://github.com/Azure/azure-sdk-for-js/pull/13162 is still open. If it is not merged by 1/14, we would like to release JS SDK for Azure Attestation out-of -band

**Comment by mssindhurid** [comment]
#sign-off

**Comment by mssindhurid** [comment]
@xiangyan99  - Can you please review this PR and merge if approved

**Review by xiangyan99 — COMMENTED** [review]
@ jeremymeng

File: releases/2021-01/js.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 ## Release highlights
 
-### _Package name_
+### Microsoft Azure Attestation [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/attestation/attestation/CHANGELOG.md)
 
-- Major changes only!
+  - Initial release of attestation Beta version to support data-plane operations of Microsoft Azure Attestation.
```
Comment (czubair) [inline releases/2021-01/js.md:39]: edit: Initial release of Microsoft Azure Attestation

**Review by czubair — APPROVED** [review]

File: releases/2021-01/js.md (hunk 1 above)
Comment (mssindhurid) [reply releases/2021-01/js.md]: @czubair  - updated per feedback

**Comment by sadasant** [comment]
@mssindhurid can I move your changes to my PR? https://github.com/Azure/azure-sdk/pull/2240/files

**Comment by sadasant** [comment]
I'll close this on favor of this other PR https://github.com/Azure/azure-sdk/pull/2240

Please submit feedback on what I've added on PR #2240. If I end up merging that PR before we get your feedback, don't worry! Please make a new PR on top of my updates and message me, and I'll make sure the changes are taken in.

**Pull request closed without merging**